packages/dev/src/utils/getClerkPackages.js

```diff
@@ -0,0 +1,25 @@
+import { readFile } from 'node:fs/promises';
+import { dirname, join, resolve } from 'node:path';
+
+import { glob } from 'glob';
```

Comment:
Nit: Can we replace its usage with https://github.com/sindresorhus/globby since we already use this in the `upgrade` package and in `scripts`? Then we're using the same package in multiple places :)

Reply:
fixed in 7616c323e2fb762ac39d3475587bfb9e141ab402